components/serverless/.gitignore

```diff
@@ -28,3 +28,6 @@ vendor/
 cover.out
 filered.cov
 log_config.yaml
+
+# Directory with temporary files used to do development
+hack/test_files
```

Comment:
Where and why we use this directory?

Reply:
I was using it to keep local test files.